mlflow/sagemaker/__init__.py

```diff
@@ -241,9 +246,12 @@ def deploy(app_name, model_path, execution_role_arn=None, bucket=None, run_id=No
                      AWS console or the ``UpdateEndpointWeightsAndCapacities`` function defined in
                      https://docs.aws.amazon.com/sagemaker/latest/dg/API_UpdateEndpointWeightsAndCapacities.html.
 
-    :param archive: If True, any pre-existing SageMaker application resources that become inactive
-                    (i.e. as a result of deploying in ``mlflow.sagemaker.DEPLOYMENT_MODE_REPLACE``
-                    mode) are preserved. If False, these resources are deleted.
+    :param archive: If ``True``, any pre-existing SageMaker application resources that become
```

Comment:
What resources are preserved, btw?

Reply:
Still question

Reply:
`archive ` will preserve SageMaker models and endpoint configurations associated with the old version of the application endpoint. I've updated the documentation to reflect this information.